express-app/models/estudio.js

```diff
@@ -14,30 +23,32 @@ var Familia = require('./familia.js');
  *
  *  Attributes:
  *  -----------
- *  OPCIONES_STATUS : tuple(tuple())
+ *  opcionesStatusChoices : array (This atribute is out of the class)
  *      The options for the current status of a study.
  *  capturista : ForeignKey
  *      The relation to the capturista that filled the study.
  *  familia : OneToOneField
  *      The family of which the study is about.
+ *  seccion : Seccion
+ *      Embedded Document of array of seccions.
+ *  respuestas : Respuesta
+ *      Array embedded document of respuestas
  *  status : TextField
  *      The study can be in several states depending if it has been approved,
  *      is on revision, has been rejected, is a draft or was deleted.
- *  numero_sae : TextField
- *      Todo: more information on this field. It appears to be some sort of
- *      id for studies (refer to the sample study provided by the stakeholder).
  */
 class Estudio extends Document {
     constructor() {
         super();
 
-        this.capturista = User;
+        this.capturista = Number;
```

Comment:
is this attribute the id from the User model? Don't you think we need all the User model?